questions/wasteful_events.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+title: "Wasteful events"
+parent: Katas
+---
+Some of the Developers see Scrum events as a waste of time because nothing comes out of them. The team seem to have the events because the Scrum Guide says so. How would you approach this situation as the Scrum Master?
```

Comment:
```suggestion
Some of the Developers see Scrum events as a waste of time because nothing comes out of them. The team seems to hold the events because the Scrum Guide says so. How would you approach this situation as the Scrum Master?
```